### PR TITLE
RN-464: Fix alias transforms not loading properly in viz-builder

### PR DESCRIPTION
**Pull request by rohan-bes**
### Issue RN-464:

Copy of: https://github.com/beyondessential/tupaia/pull/3847

### CODE REVIEW

File: packages/admin-panel/src/VizBuilderApp/components/DataLibrary/TransformDataLibrary.js [hunk 1]
```diff
@@ -100,15 +100,19 @@ export const TransformDataLibrary = ({ transform, onTransformChange, onInvalidCh
...
+TransformDataLibrary.defaultProps = {
+  transform: [],
+};
+
 TransformDataLibrary.propTypes = {
   transform: PropTypes.oneOfType([
     PropTypes.arrayOf(
       PropTypes.shape({
-        transform: PropTypes.string.isRequired,
+        transform: PropTypes.string,
```
[the remark this answers is not in the document]
Comment (rohan-bes) [reply packages/admin-panel/src/VizBuilderApp/components/DataLibrary/TransformDataLibrary.js:111]: That makes sense, I'll add it back 👍

File: packages/admin-panel/src/VizBuilderApp/components/DataLibrary/AggregationDataLibrary.js [hunk 2]
```diff
@@ -11,14 +11,14 @@ import { AggregateSelectedOptionWithJsonEditor } from './component';
 
 // Converts internal value array to Viz config.aggregate data structure
 const aggregateToValue = aggregate =>
-  aggregate.map(({ id, type: code, ...restOfConfig }, index) => ({
+  aggregate.map(({ type: code, ...restOfConfig }, index) => ({
     id: `${code}-${index}`, // id used by drag and drop function
```
[the remark this answers is not in the document]
Comment (rohan-bes) [reply packages/admin-panel/src/VizBuilderApp/components/DataLibrary/AggregationDataLibrary.js:15]: I have to admit I actually didn't write this code, just ferrying through @billli0's PR. But yeah, I think the intention is to generate an `id` intially, and then keep re-using that if it already exists (which the spread will achieve)

[the remark this answers is not in the document]
Comment (edmofro) [reply packages/admin-panel/src/VizBuilderApp/components/DataLibrary/AggregationDataLibrary.js:15]: Heh yeah true. Can we make that a bit more obvious? I think it'd be lost on me that the id being generated is only used if one doesn't already exist, without telegraphing that e.g.
```suggestion
  aggregate.map(({ id, type: code, ...restOfConfig }, index) => ({
    id: id || `${code}-${index}`, // id used by drag and drop function
```

**Review by edmofro — APPROVED** [review]
Thanks for ferrying + fixing!

**Pull request merged**